app/editor/src/components/grid/Grid.tsx

```diff
@@ -0,0 +1,135 @@
+import React from 'react';
+import { FaBackwardFast, FaCaretLeft, FaCaretRight, FaForwardFast } from 'react-icons/fa6';
+import { IPaged, Text } from 'tno-core';
+
+import { SortAction, SortDirection } from './SortAction';
+import * as styled from './styled';
+
+interface IGridHeaderColumnProps {
+  name?: string;
+  label?: React.ReactNode;
+  sortable?: boolean;
+}
+
+interface IGridProps<T> {
+  className?: string;
+  data: IPaged<T>;
+  renderHeader?: () => (React.ReactNode | IGridHeaderColumnProps)[];
+  showPaging?: boolean;
+  renderRow: (row: T) => React.ReactNode[];
+  onNavigatePage?: (page: number) => void;
+  onQuantityChange?: (quantity: number) => void;
+  onSortChange?: (column: IGridHeaderColumnProps, direction: SortDirection) => void;
+}
+
+export const Grid = <T,>({
```

Comment:
A very simple component for grid tables.  To replace all the FlexTable components.